pkgs/top-level/all-packages.nix

```diff
@@ -23151,8 +23151,16 @@ with pkgs;
 
   libtifiles2 = callPackage ../development/libraries/libtifiles2 { };
 
-  libtiff = callPackage ../development/libraries/libtiff { };
-  libtiff_4_5 = callPackage ../development/libraries/libtiff/4.5.nix { };
+  inherit
```

Comment:
Doing 3 callPackage calls and then inherit is a bit pointless. We can write those out straight away.

Reply:
Sadly, we can't.  See https://github.com/NixOS/nixpkgs/pull/298561#issuecomment-2016740260 for why this indirection is required at the moment.

I also added a hint on why this is done in the commit message.

We might even add a comment there, but since I'm following the recommended solution, I guess some more such indirections will be added to `all-packages.nix` over time and we probably shouldn't add the same explanatory comment to all of them within the same file.

Reply:
I mean the CI check would just fail for this PR which maybe doesn't justify doing such dances but whatever.